2/debian-10/rootfs/opt/bitnami/scripts/libopenldap.sh

```diff
@@ -376,13 +376,13 @@ ldap_initialize() {
         if is_boolean_yes "$LDAP_ENABLE_TLS"; then
             ldap_configure_tls
         fi
-        if is_boolean_yes "$LDAP_SKIP_DEFAULT_TREE"; then
-            info "Skipping default schemas/tree structure"
+        if ! is_dir_empty "$LDAP_CUSTOM_LDIF_DIR"; then
+            ldap_add_custom_ldifs
         else
             # Initialize OpenLDAP with schemas/tree structure
             ldap_add_schemas
-            if ! is_dir_empty "$LDAP_CUSTOM_LDIF_DIR"; then
-                ldap_add_custom_ldifs
+            if is_boolean_yes "$LDAP_SKIP_DEFAULT_TREE"; then
+                info "Skipping default schemas/tree structure"
```

Comment:
Should this message be changed?
```suggestion
                info "Skipping default tree structure"
```

Reply:
Agreed.